front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -443,6 +447,16 @@ export default function AssistantBuilder({
                     ))}
                   </TabsList>
                 </Tabs>
+                <Button
+                  icon={
+                    isPreviewPanelOpen
+                      ? ChevronDoubleRightIcon
+                      : ChevronDoubleLeftIcon
+                  }
+                  variant="ghost"
+                  tooltip={isPreviewPanelOpen ? "Hide preview" : "Open preview"}
+                  onClick={() => setIsPreviewPanelOpen(!isPreviewPanelOpen)}
+                />
```

Comment:
<img width="1507" alt="Screenshot 2025-05-20 at 16 00 37" src="https://github.com/user-attachments/assets/9e4292c0-bbc1-4347-9c20-754f69825c3d" />

super nitpick but I feel slightly weird that this button doesn't have border bottom, what do you think of removing `gap-4` from parent div and do something like this? (we can also remove border from TabList and add a border in the top div but that is not the same alignment) 

```suggestion
                <div className="border-b border-border pl-2">
                  <Button
                    icon={
                      isPreviewPanelOpen
                        ? ChevronDoubleRightIcon
                        : ChevronDoubleLeftIcon
                    }
                    variant="ghost"
                    tooltip={
                      isPreviewPanelOpen ? "Hide preview" : "Open preview"
                    }
                    className="flex-"
                    onClick={() => setIsPreviewPanelOpen(!isPreviewPanelOpen)}
                  />
                </div>
```